### PR TITLE
Fixed calculate_max_string_length unit test to work in CLion

### PR DESCRIPTION
**Pull request by NathanJPhillips**
This removes line breaks from the WHEN descriptions (which is probably what fixes CLion compatibility).
It also makes them more accurately describe what they test and fixes some mis-formatting in them.

### CODE REVIEW

**Comment by smowton** [comment]
@romainbrenguier @allredj can't tag you for review for some reason, but this one is surely for you :)

**Review by reuk — APPROVED** [review]

File: unit/solvers/refinement/string_constraint_generator_valueof/calculate_max_string_length.cpp
Comment (thk123) [inline unit/solvers/refinement/string_constraint_generator_valueof/calculate_max_string_length.cpp:124]: I don't think this is the best way to output extra info - Catch provides [logging macros](https://github.com/philsquared/Catch/blob/master/docs/logging.md) and there is support for using `INFO` on `irept`: (see `unit_tests.cpp`). (non-blocking)

File: unit/solvers/refinement/string_constraint_generator_valueof/calculate_max_string_length.cpp
Comment (thk123) [inline unit/solvers/refinement/string_constraint_generator_valueof/calculate_max_string_length.cpp]: Nit: I don't think the extra bracket here helps anything.

**Review by thk123 — APPROVED** [review]
LGTM - couple of suggestions but should get approval from original test writer

**Comment by allredj** [comment]
I'll leave it to @romainbrenguier.

**Comment by romainbrenguier** [comment]
> I'll leave it to @romainbrenguier.

Actually @owen-jones-diffblue wrote this test, so he should be the one confirming this is fine

**Review by owen-mc-diffblue — APPROVED** [review]

File: unit/solvers/refinement/string_constraint_generator_valueof/calculate_max_string_length.cpp
Comment (NathanJPhillips) [reply unit/solvers/refinement/string_constraint_generator_valueof/calculate_max_string_length.cpp:124]: I think this is the WHEN for this test rather than just a logging message.

**Pull request merged**
